daisy_ui_components_site/mix.exs

```diff
@@ -55,8 +55,8 @@ defmodule DaisyUIComponentsSite.MixProject do
       {:dns_cluster, "~> 0.1.1"},
       {:bandit, "~> 1.2"},
       {:phoenix_storybook, "~> 0.6.0"},
-      {:daisy_ui_components, "~> 0.1.6"}
-      # {:daisy_ui_components, path: "../"}
+      # {:daisy_ui_components, "~> 0.1.6"}
+      {:daisy_ui_components, path: "../"}
```

Comment:
no need for the changes on daisy_ui_compoennts_site, specially on mix lock file. I can generate the release and update the project on another PR

Reply:
[returning to same mix](https://github.com/phcurado/daisy_ui_components/pull/31/commits/9b256dda0eda4681489334d6a6fede08c04b4f3a)